crates/meilisearch/src/routes/indexes/settings.rs

```diff
@@ -63,6 +64,12 @@ macro_rules! make_setting_route {
             use $crate::Opt;
             use $crate::routes::{is_dry_run, get_task_id, SummarizedTaskView};
 
+            #[allow(dead_code)]
+            fn verify_setting_exists() {
+                let meilisearch_types::settings::Settings { $attr: _, .. } =
+                    meilisearch_types::settings::Settings::<meilisearch_types::settings::Unchecked>::default();
+            }
```

Comment:
This doesn't fit the needs. Here, you are checking if the `meilisearch_types::settings::Settings` contains the provided setting, however that's not what I wanted, I wanted to ensure that every setting contained in the `meilisearch_types::settings::Settings` struct are specified in the `make_setting_routes` macro.

This function should be moved in the `make_setting_routes` (plural) macro (upper) instead of `make_setting_route` and the `Settings` struct should be exhaustively destructured (if it ends by `, .. }` it's wrong).